avm/res/kusto/cluster/main.bicep

```diff
@@ -370,6 +372,9 @@ output name string = kustoCluster.name
 @description('The location the resource was deployed into.')
 output location string = kustoCluster.location
 
+@description('The identity of the cluster.')
+output identity object = kustoCluster.identity
```

Comment:
You can have this output if you want. But the one that is absolutely required is this one:
```bicep
@description('The principal ID of the system assigned identity.')
output systemAssignedPrincipalId string = kustoCluster.?identity.?principalId ?? ''
```

([spec reference](https://azure.github.io/Azure-Verified-Modules/specs/shared/interfaces/#managed-identities))

Reply:
Hence the error message ;) 
![image](https://github.com/user-attachments/assets/21ce5e3b-f87f-4f36-914c-e19010f4ec9c)

If the error message did not help understand & solve the issue, please provide us with set feedback so we can work on that :) Somebody just wrote this text someday 😄

Reply:
Thanks @AlexanderSehr, I updated the spec with the correct value in a PR here `systemAssignedMIPrincipalId` as well.